### PR TITLE
Locate schema update

### PR DESCRIPTION
**Pull request by ColinKerr**

### CODE REVIEW

File: core/transformer/src/test/standalone/IModelTransformer.test.ts [hunk 1]
```diff
@@ -1101,6 +1101,10 @@ describe("IModelTransformer", () => {
     targetDbTestCopy.close();
     seedDb.close();
     setToStandalone(targetDbPath);
+    // StandaloneDb.upgradeStandaloneSchemas is the suggested method to handle a profile upgrade but that will also upgrade
+    // the BisCore schema.  This test is explicitly testing that he BisCore schema will be updated from the source iModel
```
Comment (MichaelBelousov) [inline core/transformer/src/test/standalone/IModelTransformer.test.ts:1105]: nit: `that the` instead of `that he`

**Review by MichaelBelousov — APPROVED** [review]

File: full-stack-tests/presentation/src/backend/RulesetsEmbedding.test.ts [hunk 2]
```diff
@@ -46,7 +39,13 @@ describe("RulesEmbedding", () => {
   });
 
   beforeEach(async () => {
-    imodel = createSnapshotFromSeed(testIModelName, "assets/datasets/Properties_60InstancesWithUrl2.ibim");
+    const iModelPath = path.resolve(__dirname, "RulesetEmbedding.test.bim");
+    fs.existsSync(iModelPath) && fs.unlinkSync(iModelPath);
```
Comment (MichaelBelousov) [inline full-stack-tests/presentation/src/backend/RulesetsEmbedding.test.ts:43]: nit: is this really a conditional style we encourage?

File: core/transformer/src/test/standalone/IModelTransformer.test.ts (hunk 1 above)
Comment (ColinKerr) [reply core/transformer/src/test/standalone/IModelTransformer.test.ts]: fixed

File: full-stack-tests/presentation/src/backend/RulesetsEmbedding.test.ts (hunk 2 above)
Comment (aruniverse) [reply full-stack-tests/presentation/src/backend/RulesetsEmbedding.test.ts:43]: meh its in test code

**Review by aruniverse — APPROVED** [review]

File: full-stack-tests/presentation/src/backend/RulesetsEmbedding.test.ts (hunk 2 above)
Comment (MichaelBelousov) [reply full-stack-tests/presentation/src/backend/RulesetsEmbedding.test.ts:43]: hence nit, resolve at will

**Review by grigasp — APPROVED** [review]

**Pull request merged**